project.clj

```diff
@@ -5,6 +5,8 @@
   :profiles {:cljsjs {:dependencies [[cljsjs/parinfer-codemirror "1.4.1-2"]
                                      [cljsjs/codemirror "5.31.0-0"]]}}
 
+  :source-paths ["src-cljsjs" "src-npm"]
```

Comment:
what needed this?

Reply:
Cursive.

It currently only recognizes `project.clj` and doesn't otherwise find the sources.